src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java

```diff
@@ -29,11 +29,32 @@
 import org.terasology.eventualSkills.events.SkillTrainedOwnerEvent;
 import org.terasology.gestalt.assets.ResourceUrn;
 import org.terasology.gestalt.entitysystem.event.ReceiveEvent;
+import org.terasology.notifications.events.ShowNotificationEvent;
+import org.terasology.notifications.model.Notification;
+import org.terasology.engine.logic.players.LocalPlayer;
+import org.terasology.engine.logic.players.event.LocalPlayerInitializedEvent;
+import org.terasology.engine.registry.In;
 
 @RegisterSystem(RegisterMode.AUTHORITY)
 public class SkillsAuthoritySystem extends BaseComponentSystem {
+
+    private static final String NOTIFICATION_ID = "EventualSkills:newSkill";
+    private static final long NOTIFICATION_DURATION = 10000;
+
     private static final Logger logger = LoggerFactory.getLogger(SkillsAuthoritySystem.class);
 
+    @In
+    LocalPlayer localPlayer;
+
+    private void showNewSkillNotification(String newSkillName) {
+        Notification notification =
+                new Notification(NOTIFICATION_ID,
+                        "New Skill Acquired",
+                        "Acquired Skill " + newSkillName,
```

Comment:
I think we can omit the repetition of "New Skill Acquired" in combination with "Acquired Skill".
How about 
```suggestion
                        "Today I Learned",
                        newSkillName,
```